dev/docs/styles.html

```diff
@@ -14,7 +15,17 @@
   }
 
   header {
-    background-color: #01579b;
+    background-color: white;
+    color: #424242;
+  }
+
+  nav.navbar {
+    min-height: 57px;
+    padding: 6px 0;
+  }
+
+  nav.nav.navbar-fixed-top {
```

Comment:
Done, and updated http://jcollins1.pdx.corp.google.com:8012/.  It does look much more like the one on flutter.io now.